### PR TITLE
#38 Keyboard events (Departures Screen)

### PR DESCRIPTION
**Pull request by mathvaillant**

### CODE REVIEW

**Comment by mathvaillant** [comment]
[Keyboard Events - Departures](https://trello.com/c/KpqGQE29/38-keyboard-events-departures)

File: src/screens/MainMenu.tsx [hunk 1]
```diff
@@ -43,25 +44,19 @@ export function MainMenuScreen() {
...
-      "KEYCODE_1",
-      "KEYCODE_AT",
-      "KEYCODE_2",
-      "KEYCODE_A",
-      "KEYCODE_B",
-      "KEYCODE_C",
-      "KEYCODE_0",
-      "KEYCODE_SLASH",
-      "KEYCODE_EQUALS",
+      ...hardwarekeys.numeric[0],
```
Comment (mathvaillant) [inline src/screens/MainMenu.tsx:47]: @joel-gouveia the key mapping

**Review by joel-gouveia — APPROVED** [review]
Looks Good. Great job on the key mapping

**Review by joel-gouveia — APPROVED** [review]

**Comment by mathvaillant** [comment]
@joaoqueiroz95 Waiting for your review

**Review by joaoqueiroz95 — APPROVED** [review]
Looks good to me.
The only thing, if you could would be to show the changes you made on the design. I noticed you updated

**Pull request merged**